Dockerfile

```diff
@@ -1,17 +1,25 @@
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ubuntu:24.04 AS base
+FROM ubuntu:20.04 AS base
 
 RUN useradd --system --user-group --home-dir /home/autodba --shell /bin/bash autodba
 
-RUN apt-get update
-RUN apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get install -y --no-install-recommends \
     curl            \
     jq              \
-    nodejs          \
-    npm             \
     procps          \
-    wget
+    wget            \
+    gnupg           \
+    software-properties-common
```

Comment:
As you've added this dependency to `base`, let's remove it from the `autodba` target below, i.e., https://github.com/crystaldb/autodba/blob/13703811d28dd0f4e3450a0976dd44976c56e7bb/Dockerfile#L107